src/pages/home/report/FloatingMessageCounter/index.js

```diff
@@ -11,14 +11,8 @@ import withLocalize, {withLocalizePropTypes} from '../../../../components/withLo
 import FloatingMessageCounterContainer from './FloatingMessageCounterContainer';
 
 const propTypes = {
-    /** Count of new messages to show in the badge */
-    count: PropTypes.number,
-
     /** Whether the marker is active */
-    active: PropTypes.bool,
-
-    /** Callback to be called when user closes the badge */
-    onClose: PropTypes.func,
+    isActive: PropTypes.bool,
 
     /** Callback to be called when user clicks the marker */
```

Comment:
```suggestion
    /** Callback to be called when user clicks the New Messages indicator */
```
Not sure if these comments were copy paste from the new messages marker, or if we're calling both things a marker, but I think it's good to differentiate for clarity